Sources/SwiftProtobuf/Google_Protobuf_Any+Extensions.swift

```diff
@@ -16,6 +16,8 @@
 // don't remove unless obsolete on all platforms
 import Foundation
 
+public let defaultAnyTypeURLPrefix: String = "type.googleapis.com"
```

Comment:
I moved this since this file has the only use of it in a public API (the only other file that references this constant is purely internal).